lib/core/src/Network/Ntp.hs

```diff
@@ -0,0 +1,118 @@
+{-# LANGUAGE NumericUnderscores #-}
+{-# OPTIONS_GHC -fno-warn-orphans #-}
+
+-- |
+-- Copyright: © 2020 IOHK
+-- License: Apache-2.0
+--
+-- This module provides the Ntp client related settings
+-- used in a number of places throught codebase.
+
+module Network.Ntp
+    ( ntpSettings
+    , NtpTrace (..)
+    ) where
+
+import Prelude
+
+import Cardano.BM.Data.Severity
+    ( Severity (..) )
+import Cardano.BM.Data.Tracer
+    ( DefinePrivacyAnnotation (..), DefineSeverity (..) )
+import Data.Text.Class
+    ( ToText (..) )
+import Network.NTP.Packet
+    ( Microsecond (..) )
+import Network.NTP.Query
+    ( NtpSettings (..) )
+import Network.NTP.Trace
+    ( IPVersion (..), NtpTrace (..) )
+
+ntpSettings :: NtpSettings
+ntpSettings = NtpSettings
+    { ntpServers = [ "0.de.pool.ntp.org", "0.europe.pool.ntp.org"
+                   , "0.pool.ntp.org", "1.pool.ntp.org"
+                   , "2.pool.ntp.org", "3.pool.ntp.org" ]
+    , ntpResponseTimeout = 1_000_000
+    , ntpPollDelay       = 300_000_000
+    }
+
+instance ToText IPVersion where
+    toText IPv4 = "IPv4"
+    toText IPv6 = "IPv6"
+
+instance ToText NtpTrace where
```

Comment:
Pity that NtpTrace doesn't have a Show instance or something like that to provide textual log messages.